<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Lenovo-PC" ignored="0" name="用户管理系统测试用例" tests="7" failures="2" timestamp="24 八月 2018 08:58:27 GMT" time="2.804" errors="0">
  <testcase name="loginTrue" time="1.69" classname="com.course.cases.LoginTest">
    <failure type="org.apache.http.conn.HttpHostConnectException" message="Connection to http://localhost:8888 refused">
      <![CDATA[org.apache.http.conn.HttpHostConnectException: Connection to http://localhost:8888 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:754)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:732)
	at com.course.cases.LoginTest.getResult(LoginTest.java:74)
	at com.course.cases.LoginTest.loginTrue(LoginTest.java:41)
	at com.sun.proxy.$Proxy0.invoke(Unknown Source)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:345)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 43 more
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- loginTrue -->
  <testcase name="loginFalse" time="1.054" classname="com.course.cases.LoginTest">
    <failure type="org.apache.http.conn.HttpHostConnectException" message="Connection to http://localhost:8888 refused">
      <![CDATA[org.apache.http.conn.HttpHostConnectException: Connection to http://localhost:8888 refused
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:158)
	at org.apache.http.impl.conn.AbstractPoolEntry.open(AbstractPoolEntry.java:149)
	at org.apache.http.impl.conn.AbstractPooledConnAdapter.open(AbstractPooledConnAdapter.java:121)
	at org.apache.http.impl.client.DefaultRequestDirector.tryConnect(DefaultRequestDirector.java:573)
	at org.apache.http.impl.client.DefaultRequestDirector.execute(DefaultRequestDirector.java:425)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:820)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:754)
	at org.apache.http.impl.client.AbstractHttpClient.execute(AbstractHttpClient.java:732)
	at com.course.cases.LoginTest.getResult(LoginTest.java:74)
	at com.course.cases.LoginTest.loginFalse(LoginTest.java:55)
	at com.sun.proxy.$Proxy0.invoke(Unknown Source)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:345)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at org.apache.http.conn.scheme.PlainSocketFactory.connectSocket(PlainSocketFactory.java:123)
	at org.apache.http.impl.conn.DefaultClientConnectionOperator.openConnection(DefaultClientConnectionOperator.java:148)
	... 43 more
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- loginFalse -->
  <testcase name="addUser" time="0.0" classname="com.course.cases.AddUserTest">
    <skipped/>
  </testcase> <!-- addUser -->
  <testcase name="getUserInfo" time="0.0" classname="com.course.cases.GetUserInfoTest">
    <skipped/>
  </testcase> <!-- getUserInfo -->
  <testcase name="updateUserInfo" time="0.0" classname="com.course.cases.UpdateUserInfoTest">
    <skipped/>
  </testcase> <!-- updateUserInfo -->
  <testcase name="deleteUser" time="0.0" classname="com.course.cases.UpdateUserInfoTest">
    <skipped/>
  </testcase> <!-- deleteUser -->
  <testcase name="getUserListInfo" time="0.0" classname="com.course.cases.GetUserInfoListTest">
    <skipped/>
  </testcase> <!-- getUserListInfo -->
</testsuite> <!-- 用户管理系统测试用例 -->
